js/app/access-ucl-eligibility-widget.js

```diff
@@ -186,7 +186,7 @@
                 });
 
                 var eligibilityData = {};
-                var get_url = (location.href.indexOf("micro.ucl.ac.uk") > -1?"http://micro.ucl.ac.uk/access-ucl/www/eligibility-2022.php":(location.href.indexOf("wwwapps-uat.ucl.ac.uk") > -1?"https://wwwapps-uat.ucl.ac.uk/digital-presence-services/access-ucl/www/eligibility-2022.php":"https://www.ucl.ac.uk/digital-presence-services/access-ucl/www/eligibility-2022.php"));
+                var get_url = (location.href.indexOf("local-micro.lndo.site") > -1?"https://local-micro.lndo.site/access-ucl/www/eligibility-2023.php":(location.href.indexOf("wwwapps-uat.ucl.ac.uk") > -1?"https://wwwapps-uat.ucl.ac.uk/digital-presence-services/access-ucl/www/eligibility-2023.php":"https://www.ucl.ac.uk/digital-presence-services/access-ucl/www/eligibility-2023.php"));
```

Comment:
How come it's now using the `local-micro.lndo.site` here, the other file in GitLab removed it if I remember correctly

Reply:
This is for local development. This logic allows it to work on local, UAT and production. I used to use a LAMP setup on the domain micro.ucl.ac.uk - but I've replaced this with Lando now, so anyone can use it.